lib/bindings/HTTPBindings.js

```diff
@@ -41,7 +41,8 @@ var http = require('http'),
     config = require('../configService'),
     context = {
         op: 'IOTAUL.HTTP.Binding'
-    };
+    },
```

Comment:
Should be the fix be covered by unit test?

Reply:
Test already added